doc/code_snippets/snippets/connectors/go/hello.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/tarantool/go-tarantool/v2"
```

Comment:
Given that these packages are not required for the purpose of this tutorial, is it enough to add such a note?
<img width="881" alt="image" src="https://github.com/tarantool/doc/assets/38073144/09855e79-35d5-44c8-823a-c9e6f1ad9ec6">

Reply:
I would still import them by default with:

```
    _ "github.com/tarantool/go-tarantool/v2/datetime"
    _ "github.com/tarantool/go-tarantool/v2/decimal"
    _ "github.com/tarantool/go-tarantool/v2/uuid"
```

in the example. Because the example may be the basis for some experiments, and it could be confusing if datetime will not work in such experiments. This is a common problem that I see in user feedbacks.